src/routes/console/project-[project]/functions/createVariable.svelte

```diff
@@ -16,13 +16,18 @@
 
     const dispatch = createEventDispatcher();
 
+    const close = () => {
```

Comment:
Small thing: I think we are trying to use regular functions instead of arrow functions. The rest lgtm :+1: